rfc/rfc014-authorization-credential.md

```diff
@@ -101,26 +102,48 @@ When the credential is given in the context of explicit consent, the `legalBase.
 It MUST contain a value for the `path` and `type` fields. The `path` is a relative path to the service data endpoint and the `type` contains the media type as specified by [RFC6838](https://datatracker.ietf.org/doc/html/rfc6838).
 The evidence resource MUST be accessible with an access token that was created with the corresponding credential.
 
-### 3.2.4 Restrictions
+### 3.2.4 Resources
 
-The `restrictions` object MAY be used to restrict access. The base access is provided by the policy (`purposeOfUse`) as defined by the Bolt.
-If any restrictions are added, they override the Bolt policy. 
-Restrictions MAY NOT grant more access rights than the policy.
-All entries in the `restrictions` list contain a `resource`. The `resource` fields contains the relative path to the service data endpoint. 
-The `operations` field contains a list of operations allowed on this resource. The valid options are a subset of the [FHIR specification](http://hl7.org/fhir/stu3/http.html): `read`, `vread`,`update`,`patch`,`delete`,`history (instance)`, `create` and `search`.
+The `resources` object MAY be used to extend access. The base access is provided by the policy (`purposeOfUse`) as defined by the Bolt.
+If any resources are added, they extend on the Bolt policy. 
+All entries in the `resources` list contain a `path`. The `path` field contains the relative path to the service data endpoint. 
+The `operations` field contains a list of operations allowed on this resource. The valid options are a subset of the [FHIR specification](http://hl7.org/fhir/stu3/http.html): `read`, `vread`,`update`,`patch`,`delete`,`history (instance)`, `create`, `search` and `document`.
 The `userContext` field defines if the resource requires user context. If `true`, an authentication token MUST be present in the OAuth flow. 
 
+### 3.2.6 Local parameters
+
+The `localParameters` object allows an issuer to add additional parameters to the credential.
+These parameters can help to determine access at runtime.
+The contents of `localParameters` is a custom JSON object.
```

Comment:
why not leave it completely free, so it can be a string/number/bool/whatever type the issuers which it to be?

Reply:
a custom piece of JSON provides all the freedom right? The only thing disallowed is `localParameters: 5` which would be stupid anyway.